utils/storage.js

```diff
@@ -6,20 +6,36 @@ function getUsers() {
     return jsData;
 }
 
+function setUsers(users) {
+    fs.writeFileSync('./data/users.json', JSON.stringify(users), 'utf-8');
+}
+
 function getIdeas() {
     const jsonData = fs.readFileSync('./data/ideas.json', 'utf-8');
     const jsData = JSON.parse(jsonData);
     return jsData;
 }
 
+function setIdeas(users) {
+    fs.writeFileSync('./data/ideas.json', JSON.stringify(users), 'utf-8');
+}
+
 function getDonations() {
     const jsonData = fs.readFileSync('./data/donations.json', 'utf-8');
     const jsData = JSON.parse(jsonData);
     return jsData;
 }
 
+function setDonations(users) {
```

Comment:
Should be `donations`, not `users`

Reply:
function setIdeas(users) {
    fs.writeFileSync('./data/ideas.json', JSON.stringify(users), 'utf-8');
}
Same as here ideas, not users. Right?